docs/tutorials/index.html

```diff
@@ -4,30 +4,35 @@
 include_masonry: "true"
 ---
 
-{% assign tutorials=site.pages | where: "category", "tutorials" | sort: "subject" | group_by: "subject" %}
+<p class="lead">
+    These tutorials will walk you through working with a wide range of hardware and
+    software. If you're looking for help with a specific goal, this is the place to
+    look!
```

Comment:
I think that the tutorials should generally be focused on technical things that one might want to do with ev3dev which are specific to our platform. In general, I want to keep the number of "novel but tangential" tutorials to a minimum. It's a fine line, though; for example, I really like the LIDAR and RasPi camera ones, but I would rather not have generic "robotics" tutorials like odometry as a whole. That type of thing is probably better suited for a project page.